tests/ui_tests/app/page/cluster/page.py

```diff
@@ -312,3 +316,101 @@ def delete_host_by_row(self, row: WebElement):
         self.wait_element_visible(DeleteDialog.body)
         self.find_and_click(DeleteDialog.yes)
         self.wait_element_hide(DeleteDialog.body)
+
+
+@dataclass
+class ComponentsHostRowInfo:
+    """Information from host row about host on Components page"""
+
+    name: str
+    components: str
```

Comment:
I guess it'll look better at the top of the file.
Then file structure will be: 
1. Data we are working with
2. Methods (classes, functions) that work with this data

Reply:
moved